d2core/d2map/animated_composite.go

```diff
@@ -10,9 +10,11 @@ import (
 // AnimatedComposite represents a composite of animations that can be projected onto the map.
 type AnimatedComposite struct {
 	mapEntity
-	animationMode string
-	composite     *d2asset.Composite
-	direction     int
+	//animationMode string
```

Comment:
should this be removed or uncommented?

Reply:
I have a habit of commenting stuff out then removing it later on, I'll try to get better at that.